roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -230,12 +254,16 @@ spec:
               value: "{{ calico_iptables_backend }}"
             - name: FELIX_IPTABLESLOCKTIMEOUTSECS
               value: "{{ calico_iptables_lock_timeout_secs }}"
-# should be set in etcd before deployment
-#            # Configure the IP Pool from which Pod IPs will be chosen.
-#            - name: CALICO_IPV4POOL_CIDR
-#              value: "{{ calico_pool_cidr | default(kube_pods_subnet) }}"
+            # The default IPv4 pool to create on startup if none exists. Pod IPs will be
+            # chosen from this range. Changing this value after installation will have
+            # no effect. This should fall within `--cluster-cidr`.
+            # - name: CALICO_IPV4POOL_CIDR
+            #   value: "192.168.0.0/16"
             - name: CALICO_IPV4POOL_IPIP
               value: "{{ calico_ipv4pool_ipip }}"
```

Comment:
Maybe we should edit this default value `calico_ipv4pool_ipip` in `roles/network_plugin/calico_defaults/defaults/main.yml`.

Default is `Always`, but if we enable BPF the default will be `Never`

Reply:
Do you mean `calico_ipv4pool_ipip `, its default value is "Off"

Reply:
Yes, but on second thought, I think it's better to leave this issue for the next PR or issue. (or need to discuss)

The default values for VXLAN and IPIP mode have been changed in Kubespray for some reason. (#8434)

FYI: https://docs.tigera.io/calico/3.28/reference/configure-calico-node#configuring-the-default-ip-pools